superset-frontend/cypress-base/.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["cypress", "@typescript-eslint"],
+  "extends": [
+    "plugin:@typescript-eslint/recommended",
+    "plugin:cypress/recommended"
+  ],
+  "rules": {
+    "import/no-unresolved": 0,
```

Comment:
Some Cypress modules might not be available if you run `eslint` in the parent folder before `npm install` in `cypress-base` (which is what our CI does).